xml/security_ldap_ca.xml

```diff
@@ -34,56 +34,58 @@
-<command>-inkey <replaceable>SERVER.key</replaceable></command> \
-<command>-out <replaceable>SERVER.p12</replaceable> -name Server-Cert</command></screen>
+<screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Self-Signed-CA</command>
+&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Server-Cert
+</command>
+</screen>
+
   <para>
-   Replace <replaceable>SERVER.crt</replaceable> with the server certificate
-   and <replaceable>SERVER.key</replaceable> with the private key to be bundled.
-   Use <option>-out</option> to specify the name of the <filename>*.p12</filename>
-   file. Use <option>-name</option> to set the friendly name, which must be
-   <literal>Server-Cert</literal>.
+   Replace <replaceable>INSTANCE_NAME</replaceable> with the instance name of the directory server.
+   This is LDAP1 in the previous sections.
   </para>
  </step>
  <step>
    <para>
-     Before you can import the file into the NSS database, you need to
-     obtain its password. The password is stored in the
-     <filename>pwdfile.txt</filename> file in the
-     <filename>/etc/dirsrv/slapd-<replaceable>INSTANCE-NAME/</replaceable></filename> directory.
+    Import the CA that has signed your certificate
    </para>
+   <screen>&prompt.sudo;<command>sudo dsctl <replaceable>INSTANCE_NAME</replaceable> tls import-ca
+   /path/to/CA/in/PEM/format/CA.pem  <replaceable>NICKNAME_FOR_CA</replaceable>
+</command>
+</screen>
+<para>Replace <literal>INSTANCE_NAME</literal> with the instance name of the directory server.
+Replace <literal>/path/to/CA/in/PEM/format/CA.pem</literal> with the full path to the CA certificate file in PEM format.
+Replace <literal>NICKNAME_FOR_CA </literal> with a speaking name for the CA. </para>
  </step>
  <step>
   <para>
-   Now import the <replaceable>SERVER.p12</replaceable> file
-   into your &ds389a; NSS database:
+   Import the server certificate and the key for the certificate.
   </para>
-  <screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Self-Signed-CA</command>
-&prompt.sudo;<command>pk12util -i <replaceable>SERVER.p12</replaceable> -d /etc/dirsrv/slapd-<replaceable>INSTANCE-NAME</replaceable>/cert9.db</command></screen>
- </step>
+  <screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls import-server-key-cert
+  <replaceable>/path/to/SERVER.pem</replaceable> <replaceable>/path/to/SERVER.key</replaceable></command>
+  </screen>
+  <para> Replace <literal>INSTANCE_NAME</literal> with the instance name of the directory server.
+Replace <literal>/path/to/SERVER.pem</literal> with the full path to the server certificate in PEM format.
+Replace <literal>/path/to/SERVER.key</literal> with the full path to the server certificate key file im PEM format.
+If the certificate and key are in the same file, you must give the filename twice.
```

Comment:
I don't think it's valid for them to be in the same file ....

Reply:
To clarify , are you saying that the certificate and key must not be in the same file? If so, should this be documented explicitly?

Reply:
Any ref I can take a look at ?

Reply:
It's standard in PEM, keys and certs are seperate files. There are *some* applications that allow them in the same file, but they explicitly request and nominate that.

Reply:
OK noted , then I will just remove this existing sentence.  Will that suffice?

Reply:
Yes thanks,